packages/lesswrong/components/dialogues/DialoguesList.tsx

```diff
@@ -106,26 +99,110 @@ const styles = (theme: ThemeType) => ({
     display: 'flex',
     justifyContent: 'space-between',
     paddingRight: 10,
-    marginRight: 3
+    marginRight: 3,
+    marginLeft: 'auto',
   },
   dialogueMatchMessageButton: {
     marginLeft: 'auto',
     marginRight: 10
   },
   dialogueMatchPreferencesButton: {
     marginLeft: 8,
-    marginRight: 0 //3
+    marginRight: 0
   },
   dialogueNoMatchesButton: {
     marginLeft: 8
   },
+  topicRecommendationsList: {
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    color: theme.palette.text.primary,
+    fontSize: 'small',
+    overflow: 'hidden',
+    justifyContent: 'space-between'
+  },
+  dialogueSectionSettings: {
+    display: "flex",
+    alignItems: "end",
+  },
+  settingsButton: {
+    cursor: "pointer",
+    marginLeft: "5px",
+  },
   findDialoguePartners: {
+    paddingRight: 5,
+  },
+  explanatoryNoteBox: {
+    paddingLeft: "5px",
+    marginBottom: "15px",
+  },
+  explanatoryNote: {
+    color: theme.palette.text.dim3,
+  },
+  link: {
+    color: theme.palette.primary.main,
+    cursor: 'pointer',
+    '&:hover': {
+      color: theme.palette.primary.light,
+    }
   }
 });
 
+interface DialogueMatchRowProps {
+  rowProps: DialogueUserRowProps<boolean>; 
```

Comment:
I would not nest props like this; it can be fine to have props which are themselves objects with fields but you should only do that if there's some reason to do that.  If you just want `DialogueMatchRowProps` to be a superset of `DialogueUserRowProps`, just say `interface DialogueMatchRowProps extends DialogueUserRowProps<boolean>`

Reply:
request fast follow

doesn't seem blocking for shipping, and is finicky to unwrangle (it involves asking about some other types you wrote for the getRowProps functin, that I don't understand)

Reply:
sure, followup in another PR is fine